packages/@webex/plugin-meetings/test/unit/spec/member/index.js

```diff
@@ -29,25 +28,49 @@ describe('member', () => {
 
   describe('roles', () => {
     it('checks that processParticipant calls processRoles', () => {
-      const participant = {};
-
-      const member = new Member({});
-
       sinon.spy(member, 'processRoles');
       member.processParticipant(participant);
 
       assert.calledOnceWithExactly(member.processRoles, participant);
     });
 
     it('checks that processRoles calls extractControlRoles', () => {
-      const participant = {};
-
-      const member = new Member({});
-
       sinon.spy(MemberUtil, 'extractControlRoles');
       member.processParticipant(participant);
 
       assert.calledOnceWithExactly(MemberUtil.extractControlRoles, participant);
     });
+  });
+
+  describe('#processParticipant', () => {
+    it('checks that processParticipant calls isHandRaised', () => {
+      sinon.spy(MemberUtil, 'isHandRaised');
+      member.processParticipant(participant);
+
+      assert.calledOnceWithExactly(MemberUtil.isHandRaised, participant);
+    });
+  })
+
+  describe('#processMember', () => {
+    it('checks that processMember calls isRemovable', () => {
+      sinon.spy(MemberUtil, 'isRemovable');
+      member.processMember();
+
+      assert.calledOnce(MemberUtil.isRemovable);
```

Comment:
just curious: why are we using calledOnce here but in other test calledOnceWithExactly ?

Reply:
there are 4 arguments of this function and all other are called with participant object only so didn't add them as I just added this test case for better code coverage